src/cpp/src/whisper/pipeline_static.cpp

```diff
@@ -815,6 +915,7 @@ std::shared_ptr<ov::Model> prepare_decoder_with_past_model(std::shared_ptr<ov::M
     expose_runtime_states_as_inputs(decoder_with_past_model);
 
     decoder_with_past_model->reshape({{"input_ids", ov::PartialShape({-1, 1})}});
+    decoder_with_past_model->set_friendly_name("Model6");
```

Comment:
I believe modes get their names automatically, what's the reason to rename this particular one?

Reply:
As we're cloning model they will have the same name.
Just aligned names here with model names when we have 3 models.

Reply:
I'd give them some semantics then